spec/system/support_interface/unauthorized_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe 'A support user authenticates via DfE Sign-in but is not authorized to access the support interface' do
+  include DfESignInHelpers
+
+  scenario 'signing in unsuccessfully (without authorization)' do
+    given_i_have_a_dfe_sign_in_account
+
+    when_i_visit_the_support_interface
+    then_i_should_be_redirected_to_login_page
+
+    when_i_sign_in_via_dfe_sign_in
+
+    then_i_should_be_see_the_not_authorized_page
+    and_i_should_see_my_email_address
+    and_i_should_not_see_support_menu
+
+    when_i_click_sign_out
```

Comment:
I don't think we have to test sign out in this spec, it's already tested in the other spec.